tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -287,13 +278,13 @@ def __init__(self, suite_name, test_queue, test_source_cls, browser_cls, browser
 
         self.test_source = test_source_cls(test_queue)
 
-        self.manager_number = next_manager_number()
+        self.manager_number = index + 1
```

Comment:
I think we should revert this and the removal of `next_manager_number`, so that each thread gets a unique name. Or (even better), pass in the test type and use that name the thread like `f"TestRunnerManager-{test_type}-{index}"`

Reply:
Thanks, updated, PTAL.